catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/filter/impl/SortByImpl.java

```diff
@@ -67,4 +67,14 @@ public PropertyName getPropertyName() {
   public SortOrder getSortOrder() {
     return sortOrder;
   }
+
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder();
```

Comment:
❓ Why not just use `String.format("propertyName=%s...."?

Reply:
Since the goal was better performance, I went with StringBuilder.  https://stackoverflow.com/questions/44117788/performance-between-string-format-and-stringbuilder